pkg/model/function-common.go

```diff
@@ -15,11 +15,10 @@ const (
 	DefaultFunctionType           FunctionType = iota
 	ReusableContainerFunctionType              // Old school knative
 	NamespacedKnativeFunctionType
-	GlobalKnativeFunctionType
 	SubflowFunctionType
 )
 
-var FunctionTypeStrings = []string{"unknown", "knative-workflow" /*"reusable"*/, "knative-namespace", "knative-global", "subflow"}
+var FunctionTypeStrings = []string{"unknown", "knative-workflow" /*"reusable"*/, "knative-namespace", "subflow"}
```

Comment:
I'm not sure if we should remove this. Removing it means old YAMLs break without a helpful error message. Better to check the value when we unmarshal it and throw a helpful error if we detect `knative-global` in the YAML. At least for however many releases we like carrying deprecated features forward.